lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -632,9 +632,26 @@ const Map<String, List<String>> blockedTestsListsMapForModes =
     'treeshaking_integration.dart',
     'text_editing_integration.dart',
     'url_strategy_integration.dart',
+
+    // The following tests are blocked to reduce the load on the build bot.
```

Comment:
Is there an issue filed to track re-enabling these that you can note here?

Reply:
Filed https://github.com/flutter/flutter/issues/71583